cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1558,4 +1558,13 @@
     <description>Kafka topic to which metadata update notifications are published</description>
   </property>
 
+  <property>
+    <name>metadata.updates.publish.enabled</name>
+    <value>true</value>
+    <description>
+      Determines if metadata updates will be published to an external system (like Apache Kafka). External systems can
```

Comment:
if it is enabled, where can the user specify Kafka configurations for publishing to (e.g. broker list/zk quorum, topic, partitions.. etc).